Dockerfile.deploy

```diff
@@ -7,14 +7,18 @@ ENV LANG=en_US.utf8
 ENTRYPOINT /usr/bin/entrypoint.sh
 
 USER root
+
+RUN yum install -y gettext
 ADD nginx.conf /etc/nginx/nginx.conf
 
 COPY scripts/entrypoint.sh /usr/bin/entrypoint.sh
+COPY scripts/settings-tmpl.json /usr/bin/settings-tmpl.json
 RUN chmod +x /usr/bin/entrypoint.sh
 
 RUN rm -f /usr/share/nginx/html/*
 
 COPY dist /usr/share/nginx/html/launch
 RUN chmod a+w /usr/share/nginx/html/launch
+RUN chmod a+w /usr/share/nginx/html/launch/index.html
```

Comment:
isn't there a flag to make `chmod` recursive to the files?

Reply:
See Tip 3 in https://hackernoon.com/tips-to-reduce-docker-image-sizes-876095da3b34

Reply:
It seems that setting `+w` on `/usr/share/nginx/html/launch` is not needed. I am only keeping `+w` on `/usr/share/nginx/html/launch/index.html` then